pynumaflow/function/_dtypes.py

```diff
@@ -72,6 +74,70 @@ def loads(self) -> Ms:
         pass
 
 
+@dataclass(frozen=True)
+class MessageT:
+    """
+    Basic datatype for data passing to the next vertex/vertices.
+
+    Args:
+        key: string key for vertex;
+             special values are ALL (send to all), DROP (drop message)
+        value: data in bytes
+        event_time: event time of the message
+    """
+
+    key: str = ""
+    value: bytes = b""
+    # There is no year 0, so setting following as default event time.
+    event_time: datetime = datetime(1, 1, 1, 0, 0)
```

Comment:
Why not have the default event time be the current time?

Reply:
I see. @vigith personally I think default time should be the current time, or otherwise we keep the field as non default.

Reply:
let's fix this in a followup PR.. we need to standardize this.